dogfood/coder/main.tf

```diff
@@ -418,6 +426,10 @@ resource "docker_container" "workspace" {
   # CPU limits are unnecessary since Docker will load balance automatically
   memory  = data.coder_workspace_owner.me.name == "code-asher" ? 65536 : 32768
   runtime = "sysbox-runc"
+  # Ensure the workspace is given time to execute shutdown scripts.
+  destroy_grace_seconds = 60
+  stop_timeout          = 60
```

Comment:
Does this make sense as we actually do not stop the container but destroy it.

Reply:
I believe when a container is destroyed, it is first stopped. See motivation here https://github.com/coder/coder/issues/6174 which explains why we're setting both `destroy_grace_seconds` and `stop_timeout`.